jina/serve/executors/__init__.py

```diff
@@ -330,11 +340,18 @@ async def __acall_endpoint__(self, req_endpoint, **kwargs):
             else contextlib.nullcontext()
         )
 
-        with _summary:
-            if iscoroutinefunction(func):
-                return await func(self, **kwargs)
-            else:
-                return func(self, **kwargs)
+        if self.tracer:
+            with self.tracer.start_span(req_endpoint, context=tracing_context) as _:
```

Comment:
but does it check if it is successfull?

Reply:
The span uses a context manager so it's handled automatically. We can add a try-except block in case we want to add custom attributes, record exceptions or set specific error status codes.

Reply:
Please add a test, I do not understand what u say, I can see from this `span` if it succeeded or failed?